### PR TITLE
fix: fix ut test fail

### PR DESCRIPTION
**Pull request by zhagnlu**
#34831

### CODE REVIEW

File: internal/proxy/task_index_test.go [hunk 1]
```diff
@@ -651,6 +658,7 @@ func Test_parseIndexParams(t *testing.T) {
 		}
 		err := cit.parseIndexParams()
 		assert.NoError(t, err)
+		sortKeyValuePairs(cit.newIndexParams)
 		assert.Equal(t, cit.newIndexParams, []*commonpb.KeyValuePair{{Key: common.IndexTypeKey, Value: indexparamcheck.IndexHybrid},
```
Comment (chyezh) [inline internal/proxy/task_index_test.go:662]: just use `assert.ElementsMatch` may be better?

Comment (zhagnlu) [reply internal/proxy/task_index_test.go:662]: good method, this is also ok

**Pull request merged**